renovate.json

```
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
```

See PR newrelic/nri-kubernetes#275:
```suggestion
  "extends": [
    "config:base",
    // Disable the creation of this issue that renovate updates with pending PRs. We do that with Jira.
    // e.g.: https://github.com/newrelic/nri-kubernetes/issues/205
    ":disableDependencyDashboard"
  ],
```

---

I have added it to the last modification.